src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -503,6 +506,18 @@ public final void destroy()
      */
     public void onDestroyed()
     {
+
+        final TileEntityColonyBuilding tileEntity = this.getTileEntity();
+        final World world = colony.getWorld();
+        final Block block = world.getBlockState(this.location).getBlock();
+
+        if (tileEntity instanceof IInventory)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless "instanceof" operator; it will always return "true". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1850)